.github/workflows/code-quality.yaml

```
@@ -0,0 +1,116 @@
+          find . -name "*.yml" | grep -v "./.tekton/" | xargs yamllint --strict --config-file ./ci/yamllint-config.yaml
+
+      # In some YAML files we use JSON strings, let's check these
+      - name: Validate JSON strings in YAML files (just syntax)
+        id: validate-json-strings-in-yaml-files
+        run: |
+          type json_verify || sudo apt-get -y install yajl-tools
+          bash ./ci/check-json.sh
+
+      - name: Validate JSON files (just syntax)
+        id: validate-json-files
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:7:149: Double quote to prevent globbing and word splitting

(shellcheck)